### DISCUSSION TITLE
Please add support for Gateway 2000 E5200 in 86box

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#3569] [Ideas]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
86box support Gateway 2000 E5200

**Additional context**

Mainboard Info:
https://theretroweb.com/motherboards/s/tyan-s1833
![image](https://github.com/86Box/86Box/assets/127716019/a6c517af-7a67-4b62-90f6-cf47b8b056f9)

![1](https://github.com/86Box/86Box/assets/127716019/0df1b3dc-abcf-4f86-83cc-dd8f75f40ba4)

Mainboard bios download:
[0AANG025.zip](https://github.com/86Box/86Box/files/12334031/0AANG025.zip)
https://theretroweb.com/motherboard/bios/0aang-7507508-630c75a0b0092320695052.exe

Mainboard Chipset Info:

- Intel 440BX (Seattle)
https://theretroweb.com/chip/documentation/82443bx-intelcorporation-6439f2b0e1f91619171113.pdf
https://theretroweb.com/chip/documentation/297738-64af150a50065363559493.pdf

Computer Photo:
![1](https://github.com/86Box/86Box/assets/127716019/c2e937d2-211d-45a7-b8d9-6f35c653e532)
![2](https://github.com/86Box/86Box/assets/127716019/cd1def41-833e-4bba-ab04-c74589e58319)
![3](https://github.com/86Box/86Box/assets/127716019/ab0d2084-7f89-464d-a766-22e26dd3db92)
![4](https://github.com/86Box/86Box/assets/127716019/b5bf9bf2-e677-4aab-8e52-dddec91d26af)
![5](https://github.com/86Box/86Box/assets/127716019/f0ff65a4-529b-4022-aec2-6d3d2b198bcc)

Screenshot of Gateway 2000 E5200 running in 86box using [i440BX] Supermicro P6SBA machine config:
![1](https://github.com/86Box/86Box/assets/127716019/a31ae16f-a7c7-48f6-9ed9-6c05cc64012a)
![2](https://github.com/86Box/86Box/assets/127716019/134187ac-c2f8-4eba-a631-a6fb673662af)
![3](https://github.com/86Box/86Box/assets/127716019/eaa267de-08de-4b42-89e7-1373ba4e77df)
![4](https://github.com/86Box/86Box/assets/127716019/a22e6743-b085-4600-80df-7a64b3f207ef)
![5](https://github.com/86Box/86Box/assets/127716019/a1412515-fae7-4c6a-81ae-417864ca2884)
![6](https://github.com/86Box/86Box/assets/127716019/d1ae7e9e-7ca9-436d-8010-3ca65c97bfef)

86box CFG & NVR file:
[cfg & nvr.zip](https://github.com/86Box/86Box/files/12334040/cfg.nvr.zip)

thank you.

**Comment by pankoza2-pl** [comment]
Has Dual CPU support, which means we will need to implement I/O APIC first (and even then we will still be limited to 1 CPU)
